src/tiled/editableobject.h

```diff
@@ -60,6 +60,9 @@ class EditableObject : public QObject
 
     Q_INVOKABLE void removeProperty(const QString &name);
 
+    Q_INVOKABLE QVariantMap propertiesIncludingInherited() const;
+    Q_INVOKABLE QVariant propertyIncludingInherited(const QString &name) const;
```

Comment:
To match the wording in the preferences, maybe `resolvedProperty` and `resolvedProperties` would be short but suitable named for this?

Reply:
Sounds good to me. Would you like the Object implementation to remain inheritedPropert(y/ies), or be shifted to this new wording?

Reply:
Please rename it as well. :-)